src/Compilers/Core/Portable/DiaSymReader/Readme.md

```diff
@@ -0,0 +1,2 @@
+The content of this directory is a copy of a subset of https://github.com/dotnet/symreader/tree/master/src/Microsoft.DiaSymReader.
+The only difference is in top-level class visibility (public in DiaSymReader, internal here).
```

Comment:
I'm going to trust you on this... do we have a work item to do the setup authoring to refer to Microsoft.DiaSymReader in the future rather than including the source here?

Reply:
Yes, i'm already exploring it. https://github.com/dotnet/roslyn/pull/25201
We need to see if the extra assembly load affects RPS tests.